Sources/Plasma/PubUtilLib/plAudioCore/plSoundBuffer.h

```diff
@@ -120,6 +121,12 @@ class plSoundBuffer : public hsKeyedObject
     plAudioFileReader * GetAudioReader();   // transfers ownership to caller
     void                SetAudioReader(plAudioFileReader *reader);
     void                SetLoaded(bool loaded);
+    plSrtFileReader*    GetSrtReader() const { return fSrtReader; }  // does not transfer ownership
+    void                SetSrtReader(plSrtFileReader* reader)
+    {
+        delete fSrtReader;
+        fSrtReader = reader;
+    }
```

Comment:
To improve compile times, let's move this into the .cpp file, remove the `#include "plAudioCore/plSrtFileReader.h"`, and add a forward declaration `class plSrtFileReader;`.